workflow/schemas/repository.json

```diff
@@ -1,6 +1,7 @@
 {
   "title": "Dataset Submission Form",
   "description": "The repository submission form allows users to submit data to a data repository.",
+  "case_description": "Repository Dataset Submission",
```

Comment:
Should we be using the words "Dataset" and "Data" at all anymore, instead of "File" and "Repository"?